lib/Mojo/Date.pm

```diff
@@ -69,6 +71,16 @@ sub to_string {
     $MONTHS[$month], $year + 1900, $h, $m, $s;
 }
 
+# ($t, $f) = _modf($x)
```

Comment:
A note here: the implementation of this function can be shorter if it is okay to use POSIX::floor()

```
sub _modf { my $t = POSIX::floor($_[0]); ($t, $_[0]-$t) } 
```